int32 frameID

float64[] target_tag_phases
uint32 num_samples

float64[] phase_errors_avg
float64[] phase_errors_std
float64[] phase_errors_max

float64[] mag_spectra_avg
float64[] mag_spectra_std
float64[] mag_spectra_max